templates/header.jsx

```diff
@@ -34,7 +34,7 @@ export default function Header(props) {
       _extension && _extension.toLowerCase()
     ].filter(Boolean)
   } = props;
-  const sizedInstruction = (mobileInstruction && device.screenSize !== 'large') ?
+  const sizedInstruction = (mobileInstruction && device.isScreenSizeMax('small')) ?
```

Comment:
Should this be
```suggestion
  const sizedInstruction = (mobileInstruction && device.isScreenSizeMax('medium')) ?
```

Reply:
It was kicking in at JS medium (!large). Which is now ja small.

Small would be for single component only, where mobile narrative lives, with the strapline.
Medium would be side by side narrative or hotgraphic.

It depends on the definition of mobile in this case. 

I see the behaviour of mobileInstruction as best placed at the 1 component layout, which is where it was with the js being out of alignment.
There is an argument for calling it a mobile+tablet thing but I'm not sure it would be as useful to cover two different layout styles, as mobileInstruction is already an exception to instruction.
.